config/schedule.rb

```diff
@@ -37,3 +37,13 @@
 every 1.day, at: '1:00 pm', roles: [:prod] do # The server is in UTC, so this is 8:00 UTC
   rake " lib_jobs:renew_alma_requests"
 end
+
+# Run on production Tuesday at 10:30am EST or 11:30am EDT (after the records are published on Sunday)
+every :tuesday, at: '2:30 pm', roles: [:prod] do # The server is in UTC, so this is 8:00 UTC
```

Comment:
@carolyncole can you update the comment to `# The server is in UTC, so this is 2:30pm UTC`